packages/lib/membership/service.ts

```diff
@@ -122,7 +157,16 @@ export const updateMembership = async (
       },
       data,
     });
-    revalidateTag(getTeamsByUserIdCacheTag(userId));
+
+    teamCache.revalidate({
+      userId,
+      id: teamId,
```

Comment:
Yeah I was doing this because I had that `byUserIdTeamId` so if any changed I should revalidate that function, but since I am splitting those then I will remove this here